backend/Flask/app.py

```diff
@@ -8,7 +8,7 @@
 DATABASE = os.path.join(PROJECT_ROOT, 'db/todo_app.db')
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = "sqlite:////" + DATABASE
```

Comment:
It should be 
``` app.config['SQLALCHEMY_DATABASE_URI'] = "sqlite:////" + DATABASE ``` right? 
[link](https://flask-sqlalchemy.palletsprojects.com/en/2.x/quickstart/#a-minimal-application)

Reply:
3 slashes works fine!